docs/html/index.md

```diff
@@ -41,12 +41,14 @@ lists or chat rooms:
 - [Discourse channel][packaging-discourse]
 - [User IRC][irc-pypa]
 - [Development IRC][irc-pypa-dev]
+- `#pip` on the [PyPA Discord][discord]
```

Comment:
should we specify here this is for development only? Or maybe just drop this to and only link under `developer/` docs (this might make a bit hard to find)?

Reply:
I think that would be preferred. We don't actually want people to find it unless for development discussions. Even for most contributors it would be better to keep the discussions on GitHub instead.

Reply:
> I think that would be preferred. We don't actually want people to find it unless for development discussions. Even for most contributors it would be better to keep the discussions on GitHub instead.

it's gone: [8e8fc9b](https://github.com/pypa/pip/pull/12840/commits/8e8fc9b893a8f89b9ffb3ae9bbe8386eceb2fc5b)